.claude/skills/autonomous-dev/SKILL.md

```
@@ -91,58 +113,91 @@ devtools（Chrome DevTools MCP）で関係する画面を渡り歩いて、現
 3. **テストの十分さ** - カバーすべきケースが漏れていないか。テスト自体が正しいか
-4. **コードの意図** - WHY のコメントが必要な箇所に書かれているか（`code-intent-documentation.md`）
+4. **コードの意図** - WHY のコメントが必要な箇所に書かれているか
 5. **シンプルさ** - 不要な抽象化、過剰なエラーハンドリング、YAGNI違反がないか
-6. **型安全性** - any/unknown の安易な使用、型アサーション（as）の濫用がないか
-7. **既存コードとの一貫性** - 周辺コードのスタイルやパターンと調和しているか
+6. **型・入力の安全性** - 言語の型システムや検証機構を素通りしていないか（例: TypeScript なら any や型アサーション as の濫用）
+7. **波及先との整合性** - 呼び出し元・関連テスト・設定・ドキュメントに影響が漏れていないか。周辺コードのスタイルやパターンと調和しているか
 
 #### レビューの進め方
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)